docs/installation-guide.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Installation Guide"
+weight: 10
+---
+
+
+## AsyncAPI CLI Installation
+To use the AsyncAPI generator, you will need to install the AsyncApi generator CLI tool. The CLI tool allows you to generate message-based API boilerplate code or documentation. This guide will show you how to set up your local development environment to start using the generator library.
+
+### Pre-installation
+Before you install and use the AsyncAPI CLI, you'll need to have installed:
+1. Node.js v12.16 and higher
+2. Npm v6.13.7 and higher
+   
+To verify the versions of node and npm you have, run the following command on your terminal:
+```
+npm -v
+```
+```
+node -v
+```
+
+If you don't have either Node or Npm installed, do so using the [official node.js installer](https://nodejs.org/en/download/).
+
+If you have the right versions installed, you can proceed to the CLI installation guide below else, kindly upgrade the versions of your npm or node if lower than the versions specified above.
+
+> The generator library is tested at the moment against Node 14 and NPM 6. Using newer versions is enabled but we do not guarantee they work well. Please provide feedback on the issues.
```

Comment:
ups, we did not update readme, we actually also test on node 16 and newer npm, so we are sure that newer version work like a charm, we can remove this note